build/charts/theia/provisioning/dashboards/network_topology_dashboard.json

```diff
@@ -32,15 +32,56 @@
         "uid": "PDEE91DDB90597936"
       },
       "gridPos": {
-        "h": 35,
-        "w": 23,
+        "h": 22,
+        "w": 10,
         "x": 0,
         "y": 0
       },
+      "id": 4,
+      "options": {
+        "color": "yellow",
+        "groupByPodLabel": false,
+        "layerFour": true
+      },
+      "targets": [
+        {
+          "datasource": {
+            "type": "grafana-clickhouse-datasource",
+            "uid": "PDEE91DDB90597936"
+          },
+          "meta": {
+            "builderOptions": {
+              "fields": [],
+              "limit": 100,
+              "mode": "list"
+            }
+          },
+          "queryType": "sql",
+          "rawSql": "SELECT sourcePodName, sourcePodLabels, sourcePodNamespace, sourceNodeName, destinationPodName, destinationPodLabels, destinationNodeName, destinationServicePortName, octetDeltaCount FROM flows\nWHERE sourcePodNamespace NOT IN ('kube-system', 'flow-visibility', 'flow-aggregator')\nAND destinationPodNamespace NOT IN ('kube-system', 'flow-visibility', 'flow-aggregator')\nAND ( destinationPodName != '' OR sourcePodName != '' )\nAND octetDeltaCount != 0\nAND httpVals == ''\nAND $__timeFilter(flowEndSeconds)\nORDER BY flowEndSeconds DESC",
```

Comment:
Do we want to exclude the ns` local-path-storage`, which is created in the kind cluster by default?